controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -16,15 +16,13 @@
 GAME_DURATION_SECONDS = 150
 
 
-def recording_path(when: Optional[datetime.datetime] = None) -> Path:
-    if not when:
-        when = datetime.datetime.now()
+def recording_path() -> Path:
+    now = datetime.datetime.now()
 
-    date = when.date().isoformat()
-    name = '{}.html'.format(when.isoformat())
+    date = now.date().isoformat()
     # Windows doesn't like colons in filenames
-    name = name.replace(':', '-')
-    return Path(date) / name
+    name = str(now.isoformat()).replace(':', '-')
+    return Path(date) / name / '{}.html'.format(name)
```

Comment:
Not a criticism, but I'm curious about the usage of `str.format` here over f-strings, as we're working in Python 3.7 for this project.

Reply:
While the target for the competition is 3.7, we're still supporting 3.5 for development.